.github/actions/alembic-install-dep/action.yml

```diff
@@ -20,7 +20,7 @@ runs:
 
     - name: Cache Alembic
       id: cache-alembic
-      uses: actions/cache@v4
+      uses: actions/cache/restore@v4
```

Comment:
Is it disabling automatic cache save? So using `cache/restore` and `cache/save` is more flexible?

Reply:
Exactly